src/modules/common/components/Headroom.js

```diff
@@ -153,9 +154,11 @@ class Headroom extends React.PureComponent<PropsType, StateType> {
       scrollHeight,
       positionStickyDisabled
     } = this.props
-    const { mode, transition } = this.state
+    const {mode, transition} = this.state
     const transform = mode === UNPINNED ? -scrollHeight : 0
-    const stickyTop = Headroom.calcStickyTop(mode, height, scrollHeight)
+    if (stickyAncestor && !height) {
+      console.warn('Headroom: If you supply stickyAncestor, you must define height!')
```

Comment:
Because it doesn't break the display, but only glitches a bit... If I throw an error, the complete website would stop working. (But I also thought about throwing an error)...